src/Task.ts

```diff
@@ -4,6 +4,7 @@ import { RRule } from 'rrule';
 import { replaceTaskWithTasks } from './File';
 import { getSettings } from './Settings';
 import type { Moment } from 'moment';
+import { LayoutOptions } from 'LayoutOptions';
```

Comment:
Please import with relative path (jest fails at the moment).
```suggestion
import { LayoutOptions } from './LayoutOptions';
```